utils/docker/Dockerfile.leap.15

```diff
@@ -30,9 +30,10 @@ RUN zypper --non-interactive --no-gpg-checks refresh; \
            python-devel python3-devel valgrind-devel hwloc-devel      \
            openmpi3-devel Modules man
 
-RUN pip install --upgrade pip; \
+RUN update-ca-certificates; \
```

Comment:
This doesn't really need to be combined into the existing `RUN` line.  But if it is, the `\` should line up with the rest of the lines.

Reply:
Done